jsonschema/golangci.jsonschema.json

```
@@ -1813,7 +1813,7 @@
           "properties": {
             "scope": {
               "description": "Comments to be checked.",
-              "enum": ["declarations", "toplevel", "all"],
+              "enum": ["declarations", "toplevel", "all", "noinline"],
```

Should we sort values as in the https://github.com/tetafro/godot#config?
```suggestion
              "enum": ["declarations", "toplevel", "noinline", "all"],
```